pehlaschoollibrary/proj.android-studio/app/src/com/maq/pehlaschool/library/SelectActivity.java

```diff
@@ -232,11 +232,23 @@ public static void updateStringLocale( Context context, String locale)
             case "bengali":
                 currConfig.setLocale(new Locale("bn"));
                 break;
-            default: // Do nothing as English text is set by default
+            default: 
+                currConfig.setLocale(new Locale("en"));
                 break;
         }
         //set the locale with the updated configuration
         currRes.updateConfiguration(currConfig, currDispMetrics);
     }
 
+    public static String getLocalefromIntent(Intent localeIntent){
```

Comment:
Add comment for the function

Reply:
Added comment for the getLocalefromIntent function